src/dmd/tocvdebug.d

```diff
@@ -309,6 +304,9 @@ void cv_udt(const char* id, uint typidx)
 
 void toDebug(EnumDeclaration ed)
 {
+    if (target.os != Target.OS.Windows)
+             return;
```

Comment:
The indentation of these `return` statements is off

Reply:
Fixed.